test/controllers/microposts_controller_test.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class MicropostsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @micropost = microposts(:one)
+  end
+
+  test 'should succeed get index' do
```

Comment:
Controller test classes are divided up into a "section" per action. At the beginning of each section is a section comment, e.g. `# Test index.` The comment is followed by 'should fail...' tests, then 'should succeed...' tests. See IO/DM controllers for reference. If this wasn't clear from dev practices linked in the cheat sheet, please update that page to make it clear for future devs.